examples/prod_failures.rb

```diff
@@ -24,3 +24,4 @@ def main
 end
 
 main if __FILE__ == $PROGRAM_NAME
+aaa    =   "dfdfdfdf"
```

Comment:
Style/SpaceAroundOperators: Operator `=` should be surrounded by a single space. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)